JM/index.js

```diff
@@ -17,10 +17,10 @@ app.post('/addPerson', function(postmanRequest, postmanresponse){
 
 app.get('/getAllPeople', function(postmanRequest, postmanresponse){
     const url = 'http://java-sample-api-2020.herokuapp.com/getAllPeople'
-    data = axios.get(url)
+    axios.get(url)
         .then((response) => {
             const data = response.data
```

Comment:
Everything looks good to me. Line 22 has an unused variable 'data'. Consider removing or using.